nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -205,25 +205,38 @@ def __init__(
         else:
             self.out_proj = None
             self._feat_out = d_model
+        self.xmax = pos_emb_max_len
+        device = next(self.parameters()).device
+        seq_range = torch.arange(0, self.xmax, device=device)
```

Comment:
Settingthe device in the init means that user may not move the model after loading the model.